deploy/triage-party/deploy.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+# Export this environment variable before running this script
+echo "token path: ${GITHUB_TOKEN_PATH}"
+
+export PROJECT=k8s-skaffold
+IMAGE="gcr.io/k8s-skaffold/teaparty:$(date +%F-%s)"
+export
```

Comment:
I think you mean this:
```suggestion
export IMAGE="gcr.io/k8s-skaffold/teaparty:$(date +%F-%s)"
```

Reply:
The dockerfile is this https://github.com/google/triage-party/blob/master/Dockerfile. 

Let me actually point to latest teaparty image and in the internal docs ad an extra step to publish this image.

Reply:
nvm. I blindly copied the deploy script from  https://github.com/google/triage-party. 

I will change the `docker build -t` etc.

Reply:
I decided to delete the `deploy.sh` instead change the internal docs to point to skaffold config in the skaffold repo instead the one in the http://go/skaffold-triage-party-deploy